packages/manager/modules/pci/src/components/project/empty/empty.html

```diff
@@ -1,4 +1,4 @@
-<div class="container px-0 pci-project-empty">
+<div class="container mx-0 px-0 pci-project-empty">
```

Comment:
Isn't already defined?

ref: https://github.com/twbs/bootstrap/blob/master/scss/mixins/_grid.scss#L9-L10

Reply:
mx-0 => Horizontal margin set to 0, instead of auto ;)